src/pi-gen-config.ts

```diff
@@ -25,7 +25,8 @@ export interface PiGenConfig {
   stageList: string[]
   useQcow2: string
   enableNoobs: string
-  exportLastStageOnly: string
+  exportLastStageOnly: string,
+  dockerOpts: string
```

Comment:
If the parameter is optional, reflect this in the interface:
```suggestion
  dockerOpts?: string
```
Then you can also drop the default value below and add a distinct check for a `undefined` where the value is concatenated.